openapiart/bundler.py

```diff
@@ -391,6 +391,23 @@ def _validate_required_responses(self):
                 missing_paths += "{}\n".format(error_message)
         if len(missing_paths) > 0:
             raise Exception(missing_paths)
+
+        # There must be the Error structure in the yaml which should have required kind and errors
+        err_schema = self._get_parser("$..Error").find(self._content)
+        if len(err_schema) == 0:
+            raise Exception("default must have a error schema")
```

Comment:
rephrase: default response must use Error schema

Reply:
BTW, where are we checking that `default` response is pointing to error schema ? if we're not, then validation isn't complete.

I believe currently we're only checking if Error schema exists (in which case I would rephrase the error message above even further)

Reply:
I rephrased the exception a bit , do you have a better suggestion ?

Reply:
Multiple concerns:
- We should get a singe match for `Error` as you won't have multiple of them defined; so why aren't we asserting that ? (i.e. exactly one schema MUST exist)
- If "required" is not in error schema, then it's an error (currently it's leaking through)
- Shouldn't we be validating that all default responses actually point to Error schema ?

Reply:
agreed for first two points, added checks and unit test for the same
for the last point default schema may not directly point to error schema and might have intermediate structures right